x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/multi_field_selector.tsx

```diff
@@ -220,12 +220,7 @@ export const SelectedFieldMappings: React.FC<SelectedFieldMappingsProps> = ({ is
                 type: 'icon',
               },
             ],
-            name: i18n.translate(
-              'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.fields.actions',
-              {
-                defaultMessage: 'Actions',
-              }
-            ),
+            name: '',
```

Comment:
For accessibility reasons, a table column should always have a header. You could re-name this 'Action', but removing the table header makes it harder for people using screen readers to navigate.,

Reply:
Ok, I'll put the label back. cc @julianrosado

Reply:
That's fine!